src/main/java/com/epam/gym/actuator/ExternalServiceHealthIndicator.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.gym.actuator;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExternalServiceHealthIndicator implements HealthIndicator {
+
+    @Override
+    public Health health() {
+        boolean serviceUp = checkExternalServiceHealth();
+
+        if (serviceUp) {
+            return Health.up().withDetail("External Service", "Available").build();
+        } else {
+            return Health.down().withDetail("External Service", "Not Available").build();
+        }
```

Comment:
Let's rename `serviceUp` to `isServiceUp` to increade code readability:
```suggestion
        boolean isServiceUp = checkExternalServiceHealth();

        if (isServiceUp) {
            return Health.up().withDetail("External Service", "Available").build();
        } else {
            return Health.down().withDetail("External Service", "Not Available").build();
        }
```

Please introduce String constants for duplicated String literals.